rocAL/rocAL_pybind/example/coco_pipeline.py

```diff
@@ -175,7 +175,8 @@ def __next__(self):
         self.img_size = np.zeros((self.bs * 2), dtype="int32")
         self.loader.GetImgSizes(self.img_size)
         print("Image sizes:", self.img_size)
-        #img = torch.from_numpy(self.out)
+        # Draw images: Make sure to uncomment the below line to dump images
+        img = torch.from_numpy(self.out)
```

Comment:
Can you please update the changes from my branch for this. Basically we need to make it optional based on a flag and do the normalization correct to view output properly.

Reply:
Done